assets/components/MainMenu.svelte

```diff
@@ -184,80 +198,97 @@ function renderUnread(conversation, max = 60) {
 
   <nav class="sidebar-left__nav" class:is-filtering="{filter.length > 0}" bind:this="{navEl}" on:click="{onNavItemClicked}">
     <h3>{$l('Conversations')}</h3>
-    {#if !$user.connections.size}
-      <Link href="/settings/connections">
-        <Icon name="exclamation-circle"/>
-        <span>{$l('No conversations')}</span>
-      </Link>
-    {/if}
-    {#each $user.connections.toArray() as connection}
-      <Link href="{connection.path}" class="{conversationClassNames(connection, connection)}">
+      {#if !$user.connections.size}
+        <Link href="/settings/connections">
+          <Icon name="exclamation-circle"/>
+          <span>{$l('No conversations')}</span>
+        </Link>
+      {/if}
+      {#each $user.connections.toArray() as connection}
+      <Link href="{connection.path}" class="{conversationClassNames(connection, connection)}" >
         <Icon name="network-wired"/>
-        <span>{connection.name || connection.connection_id}</span>
+        <span >{connection.name || connection.connection_id}</span>
         <span class="tooltip">{$l(connection.frozen)}</span>
-        <b class="badge" hidden="{!connection.unread}">{renderUnread(connection)}</b>
+        <b class="badge-connection" hidden="{!connection.unread}" on:click={e => {
+          toggleSection(connection.connection_id);
+          }}>{renderUnread(connection)}
+        </b>
+        <!-- Here's an icon indicating collapsed/expanded state -->
+        <div class="chevron"><Icon
+          name={collapsedStates[connection.connection_id] ? 'chevron-right' : 'chevron-down'}
+          on:click={e => {
+          toggleSection(connection.connection_id);}}/>
+        </div>
```

Comment:
This section has a bit strange formatting. Here is my suggestion:

```html
<b class="badge-connection" hidden="{!connection.unread}" on:click={() => toggleSection(connection.connection_id)}}>{renderUnread(connection)}</b>
<!-- Here's an icon indicating collapsed/expanded state -->
<div class="chevron">
  <Icon name={collapsedStates[connection.connection_id] ? 'chevron-right' : 'chevron-down'}
    on:click={e => toggleSection(connection.connection_id)}/>
</div>

Reply:
```suggestion
        <b class="badge-connection" hidden="{!connection.unread}" on:click={e => { toggleSection(connection.connection_id); }} on:keydown={e => { if (e.keyCode === 13) { toggleSection(connection.connection_id); } }}>{renderUnread(connection)}</b>
        <!-- Here's an icon indicating collapsed/expanded state -->
        <div class="chevron">
          <Icon name={collapsedStates[connection.connection_id] ? 'chevron-right' : 'chevron-down'}
          on:click={e => { toggleSection(connection.connection_id);}}/>
        </div>
```